randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -1040,7 +1042,12 @@ Extra - room_id: [28]
   > Pickup (Energy Tank)
       All of the following:
           Speed Booster
-          Space Jump or Can Activate Pillar
+          Any of the following:
+              Space Jump or Can Activate Pillar
+              # Wall Jump and Shinespark straight up - https://youtu.be/Pvv2dYRZAj4
+              Wall Jump (Intermediate)
```

Comment:
With 
1) this shinespark (probably) not being the intended path
2) the timer being so strict that you don't have a lot of leeway on the shinespark from the ground
3) the timer being strict-ish that you dont have too much leeway if you shinespark from the air

this should be an advanced shinespark